src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/config/ObjectMapperConfiguration.kt

```diff
@@ -16,7 +16,7 @@ class ObjectMapperConfiguration {
 
   @Bean
   @Primary
-  fun objectMapper(): ObjectMapper? {
```

Comment:
This looks like it's always going to return an ObjectMapper
 but there again I don't really get Kotlin yet

Reply:
Yes we can drop the ?, it comes in from copying Java code into the project.